packages/multimedia/libbluray/package.mk

```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="libbluray"
-PKG_VERSION="1.1.2"
-PKG_SHA256="a3dd452239b100dc9da0d01b30e1692693e2a332a7d29917bf84bb10ea7c0b42"
+PKG_VERSION="1.2.0"
```

Comment:
needs update at kodi that didn't happend yet https://github.com/LibreELEC/LibreELEC.tv/pull/4299